src/gui/modalMenu.cpp

```diff
@@ -140,7 +140,8 @@ bool GUIModalMenu::preprocessEvent(const SEvent &event)
 			std::string label = wide_to_utf8(getLabelByID(hovered->getID()));
 			if (label.empty())
 				label = "text";
-			message += gettext(label) + ":";
+			message += gettext(label.c_str());
+			message += ":";
```

Comment:
Why?

Reply:
`message` is of type `std::string`, and the type returned by `gettext(...)` is `char*`. 
And since we are adding `":"` just after, the compiler was considering that constant also as `char*`. But adding two `char*` is not supported. 
The conversion to `std::string` comes with the `+=` assignation
So this was the simplest way to just force the conversion of the two values to `std::string`.
Note that it was not needed before because `gettext` was just a macro returning the original type that was `std::string`.

Reply:
Ok, but then what is wrong with `gettext(label) + ":"`?

Reply:
that's where the problem is: we try to do: `std::string += ( char* + char* )` the compiler tries to solve the left operand before the assignation. So, it tries to sum two char pointers.

Reply:
I could have written `std::string(gettext(label)) + std::string(":")`, but it's probably less readable than a new line ;)